app/routes/dashboard/index.tsx

```diff
@@ -38,6 +38,10 @@ const Page = () => {
     navigate($path('/onboarding'));
   }
 
+  useEffect(() => {
```

Comment:
Why imperative api instead of component? Looks like button should hide automatically

Reply:
Hm, I'll take a look at that approach, thanks!